engine/src/main/java/org/pentaho/di/trans/DefaultTransManager.java

```diff
@@ -0,0 +1,43 @@
+package org.pentaho.di.trans;
+
+import org.pentaho.di.core.exception.KettleException;
+import org.pentaho.di.core.logging.LogChannelInterface;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+
+public class DefaultTransManager implements TransMananger {
```

Comment:
javadoc class comment for DefaultTransManager

Reply:
addressed